napari/components/viewer_model.py

```diff
@@ -722,12 +722,16 @@ def open_sample(
         KeyError
             If `plugin` does not provide a sample named `sample`.
         """
-        from ..plugins import _sample_data, available_samples
+        from ..plugins import plugin_manager
 
         try:
-            data = _sample_data[plugin][sample]['data']
+            data = plugin_manager._sample_data[plugin][sample]['data']
         except KeyError:
-            samples = available_samples()
+            samples = plugin_manager.available_samples()
+            msg = trans._(
+                f"Plugin {plugin!r} does not provide sample data "
+                f"named {sample!r}. "
```

Comment:
```suggestion
                f"Plugin {plugin!r} does not provide sample data named {sample!r}. ",
                deferred=True,
```

Translated strings need to be on a single line, see [this comment](https://github.com/napari/napari/pull/2609#issuecomment-827221281). @goanpeca could you create that issue please? 😊